### ISSUE TITLE
Hoodwink's Aghanim ability "Decoy" bug

### ISSUE THREAD

**Issue by tomochan9sai** [ValveSoftware/Dota2-Gameplay#3502]
### Ability name

Decoy

### Description

 When using Decoy as Hoodwink if you don't have enough mana for sharpshooter (ult) the decoy won't use the ability and instead just walk forward.

### Example Match ID (and possibly Timestamp)

Demo hero

**Comment by TheUnholyWrath** [comment]
#2532 #2360

**Comment by tomochan9sai** [comment]
oh i searched and didn't see those, should i just delete or close issue?

**Comment by TheUnholyWrath** [comment]
Yeah if you close this and give a thumbs up reaction to #2352 , comment anything additional underneath that would be the most you could do to increase the discoverability of this issue

**Comment by tomochan9sai** [comment]
ok, thanks
